src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -586,6 +585,26 @@ def _configure_security_policy_storage_params(cmd):
 
 
 #####
+#           sql server ad-admin
+######
+
+
+with ParametersContext(command='sql server ad-admin') as c:
+    c.argument('server_name', options_list=('--server-name', '-s'),
+               help='The name of the SQL Server')
+    c.argument('login', options_list=('--display-name', '-u'),
+               help='Display name of the Azure AD administrator user or group.')
+    c.argument('sid', options_list=('--object-id', '-i'),
+               help='The unique ID of the Azure AD administrator ')
+    c.ignore('tenant_id')
+
+
+with ParametersContext(command='sql server ad-admin create') as c:
+    c.expand('properties', ServerAzureADAdministrator, patches={
+        'tenant_id': patch_arg_make_optional})
```

Comment:
this help expend the properties and show the other parameter to the user, display-name and object id